docs/application-connector/docs/034-gs-get-client-certificate.md

```diff
@@ -62,9 +48,9 @@ A successful call returns the following response:
 {
     "csrUrl": "{CSR_SIGNING_URL_WITH_TOKEN}",
     "api":{
-        "metadataUrl":      "https://gateway.{CLUSTER_NAME}.kyma.cluster.cx/{REMOTE-ENVIRONMENT-NAME}/v1/metadata/services",
-        "eventsUrl":        "https://gateway.{CLUSTER_NAME}.kyma.cluster.cx/{REMOTE-ENVIRONMENT-NAME}/v1/events",
-        "certificatesUrl":  "https://connector-service.{CLUSTER_NAME}.kyma.cluster.cx/v1/remoteenvironments/{RE_NAME}",
+        "metadataUrl":      "https://gateway.{CLUSTER_NAME}/{REMOTE-ENVIRONMENT-NAME}/v1/metadata/services",
+        "eventsUrl":        "https://gateway.{CLUSTER_NAME}/{REMOTE-ENVIRONMENT-NAME}/v1/events",
+        "certificatesUrl":  "https://connector-service.{CLUSTER_NAME}/v1/remoteenvironments/{RE_NAME}",
```

Comment:
Sometimes you use `{REMOTE-ENVIRONMENT-NAME}` and sometimes `{RE_NAME}` which may be confusing.